week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -115,4 +150,4 @@ const books = [
   }
 ];
```

Comment:
Well done here Laetitia, in the instruction you have been asked to use the DOM method to create every element. Try it again and use the necessary DOM elements.